### PR TITLE
update wombat assets

### PR DESCRIPTION
**Pull request by oconn**
- Updated wombat assets
- Fixed join hover state bug
- Fixed poison bug

### CODE REVIEW

File: src/cljs/wombats_web_client/constants/colors.cljs [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
-               {:color-text "green" :color-hex "#419C3F"}
-               {:color-text "orange" :color-hex "#E79300"}
-               {:color-text "purple" :color-hex "#AE18E2"}
-               {:color-text "red" :color-hex "#D40000"}
-               {:color-text "white" :color-hex "#C9C9C9"}
-               {:color-text "yellow" :color-hex "#E1E700"}])
+(defonce colors-8 [{:color-text "gray" :color-hex "#454545"}
+                   {:color-text "blue" :color-hex "#2221e9"}
+                   {:color-text "green" :color-hex "#469000"}
+                   {:color-text "orange" :color-hex "#EC5A00"}
```
Comment (emilyseibert) [inline src/cljs/wombats_web_client/constants/colors.cljs]: where did you get these hex values?

Comment (oconn) [inline src/cljs/wombats_web_client/constants/colors.cljs]: Digital color meter used on the new assets

File: resources/spritesheet.tps [hunk 2]
```diff
@@ -6,7 +6,7 @@
         <key>texturePackerVersion</key>
         <string>4.3.3</string>
         <key>fileName</key>
-        <string>spritesheet.tps</string>
+        <string>/Users/mattoconnell/Projects/wombats-game/wombats-web-client/resources/spritesheet.tps</string>
```
Comment (emilyseibert) [inline resources/spritesheet.tps]: dis is wrong

File: src/cljs/wombats_web_client/constants/colors.cljs (hunk 1 above)
Comment (emilyseibert) [inline src/cljs/wombats_web_client/constants/colors.cljs]: 👍

**Comment by emilyseibert** [comment]
wooo new assets 🍾

**Pull request merged**